pkg/sfu/forwarder.go

```diff
@@ -55,39 +55,39 @@ func (v VideoPauseReason) String() string {
 // -------------------------------------------------------------------
 
 type VideoAllocation struct {
-	pauseReason         VideoPauseReason
-	isDeficient         bool
-	bandwidthRequested  int64
-	bandwidthDelta      int64
-	bandwidthNeeded     int64
-	bitrates            Bitrates
-	targetLayers        VideoLayers
-	requestLayerSpatial int32
-	maxLayers           VideoLayers
-	distanceToDesired   float64
+	PauseReason         VideoPauseReason
+	IsDeficient         bool
+	BandwidthRequested  int64
+	BandwidthDelta      int64
+	BandwidthNeeded     int64
+	Bitrates            Bitrates
+	TargetLayers        buffer.VideoLayer
+	RequestLayerSpatial int32
+	MaxLayers           buffer.VideoLayer
+	DistanceToDesired   float64
```

Comment:
Some of these fields were used by `StreamAllocator`. So, had to make them upper case. Made all of them upper case for consistency. At some point will have a re-look to make this interface cleaner, i. e. return only what stream allocator needs?